### ISSUE TITLE
Pro dashboard is very unclear about what the toggle switches do

### ISSUE THREAD

**Issue by cpaelzer** [canonical/ubuntu.com#12795]
## Summary

The descriptions on the [pro dashboard](https://ubuntu.com/pro/dashboard) are misleading. Our own filed team didn't realize that these are meant to control the services that the contract server sends to the client as "recommended to enable" when attaching a system.

## Process

1. go to https://ubuntu.com/pro/dashboard
2. log in with a user that has contracts assigned
3. check below your token the services enabled
4. try to understand (if you'd not know anything) what those toggle switches will do

## Current and expected result

Current:
The text says just "included", does that mean "part of what I pay", or "things that are included in my contract". If so why can I toggle them here. To make it worse, the service and support shows up here, which obviously can't be toggled. Overall very confusing.

Expected:
- Replace "included" with something like, "The following services are available to your subscription, enabling them here controls if they shall be enabled by default when attaching".

Others will be much better to bike-shed this into a better text, but just "included" isn't helping.

## Screenshot

Currently a contract without support:
![image](https://user-images.githubusercontent.com/14993822/231960957-b5f3be4e-fec4-4c7c-b4aa-5287e1b02a4d.png)
Currently a Contract with support:
![image](https://user-images.githubusercontent.com/14993822/231961112-2fdaaacd-598e-4a7f-999e-1df45ac29274.png)

## Browser details

Seen the same in chrome, chromium and firefox - that isn't browser specific IMHO.

**Comment by cpaelzer** [comment]
Highlighting @lechsandecki as well as @orndorffgrant / @renanrodrigo / @lucasmoura  for awareness.

**Comment by cpaelzer** [comment]
I'm not sure if the rendering of this might actually be a task of contract server, also highlighting @alnvdl just to be sure

**Comment by lechsandecki** [comment]
agreed. thanks for the feedback, @cpaelzer. is that already on your radar @leiaru-c and @lidia-luna ?

**Comment by lidia-luna** [comment]
yes, we are hoping that dashboard re-design will be made a roadmap item next cycle so we can focus on this and other dashboard issues

**Comment by carkod** [comment]
Is this still relevant? @leiaru-c @lechsandecki

**Comment by cpaelzer** [comment]
> Is this still relevant? @leiaru-c @lechsandecki

At least as it looks on https://ubuntu.com/pro/dashboard as of now it is not yet fixed.

**Comment by leiaru-c** [comment]
Hi @carkod and @cpaelzer, we are currently implementing the new designs that ar removing the toggles amongst other improvements. You can follow the progress of the task here: https://warthogs.atlassian.net/browse/WD-5646?atlOrigin=eyJpIjoiZGZjNzBiOTk0OTM1NDM0Yjg1ZmJhZTQxYTdjOTJhMzgiLCJwIjoiaiJ9 

You can find the Figma file for the design update in the ticket as well. Let me know if you have any questions!

**Comment by cpaelzer** [comment]
@leiaru-c you are right that you are removing the boxes, but now I'm even more concerned :-/
Where would a user in the future tweak which services will be default enabled?
The Figma file I've seen has the old boxes on the left and various examples without on the right. But none shows any way to tweak the default behavior of my accounts token anymore.

**Comment by petesfrench** [comment]
Hi @cpaelzer, I am checking in to see if the changes made to the Pro dashboard have addressed you concerns?
